backend/src/vscode-youi-events.ts

```diff
@@ -56,26 +56,31 @@ export class VSCodeYouiEvents implements YouiEvents {
         VSCodeYouiEvents.installing = false;
         
         if (success) {
-            let addToWorkspaceButton: any;
-            let openInNewWorkspaceButton: any;
+            const addToWorkspace: string = "Add to Workspace";
+            const openInNewWorkspace: any = "Open in New Workspace";
+            const items: string[] = [];
```

Comment:
I think thi doesn't work in theia. keep passing undefined,undefined.

Reply:
It does not work with single or double undefined too

Reply:
I check in theia and both or single buttons are displayed
The only problem is messages without buttons

Reply:
heelo world message must work. in Theia. We can check it with simple vscode ext only showing message. if not working then we can open issue to theia.

Reply:
it does not work, i have already checked

Reply:
or might be it does not work only in this environment

Reply:
clone https://github.com/tomer-epstein/vscode-add-Root-Folder.git

change: vscode.commands.executeCommand("workbench.action.addRootFolder");
to: vscode.window.showInformationMessage('Hello World!'); 

it works.

![image](https://user-images.githubusercontent.com/57438361/78222478-fc67cc00-74cd-11ea-9982-24ba07bdf399.png)

Reply:
works good - 
both
const items: string[] = [];
and
vscode.window.showInformationMessage(infoMessage, ...items)